Starcounter.Uniform.Tests/Queryables/QueryableFilterTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using Starcounter.Uniform.Generic.FilterAndSort;
+using Starcounter.Uniform.Queryables;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Starcounter.Uniform.Tests.Queryables
+{
+    public class QueryableFilterTests
```

Comment:
there should be tests that check filtering by multiple properties as well as ordering. And mix of those

Reply:
Done.